src/lib/stores.ts

```diff
@@ -15,15 +19,68 @@ export const inSnsMenu = writable<boolean>(false);
 /* === Components === */
 export const language = writable<'en' | 'es' | 'ja' | 'ru'>('en');
 export const selectedAsset = writable<Asset>(new Asset(AssetType.ICP));
-export const selectedSns = writable<string>('BOOM DAO');
-export const snsPrincipal = writable<string>('xomae-vyaaa-aaaaq-aabhq-cai');
-export const encodedSnsIcrcAccount = writable<string>('-/-');
 export const user = writable<User | undefined>(undefined);
 export const canisters = writable<Canisters | undefined>(undefined);
 export const waterNeuronInfo = writable<WaterNeuronInfo | undefined>(undefined);
 
+/* === SNS === */
+function createSns() {
```

Comment:
I feel like the name is poor, could be `createBoomerangSnsStore`?

Reply:
done.